api/src/main/java/javax/servlet/AsyncContext.java

```diff
@@ -36,84 +36,82 @@
  * {@link #dispatch} methods, call {@link #complete}.</li>
  * </ol>
  *
- * @since Servlet 3.0
+ * @since 3.0
  */
 public interface AsyncContext {
 
     /**
      * The name of the request attribute under which the original request URI is made available to the target of a
      * {@link #dispatch(String)} or {@link #dispatch(ServletContext,String)}
      */
-    static final String ASYNC_REQUEST_URI = "javax.servlet.async.request_uri";
+    String ASYNC_REQUEST_URI = "javax.servlet.async.request_uri";
```

Comment:
These are good changes, but I'm not so sure they should be included in a PR that is substantially about establishing the new project name.